policy.go

```diff
@@ -73,21 +73,21 @@ type policyData struct {
 func hashAlgToGoHash(hashAlg tpm2.AlgorithmId) hash.Hash {
 	knownDigest, isKnown := knownDigests[hashAlg]
 	if !isKnown {
-		return nil
+		panic("Unknown digest algorithm")
```

Comment:
Is panicking really better than returning an error to be handled by caller?

Reply:
I've adjusted both this function and getDigestSize to panic - getDigestSize did return an error before but it was ignored in every case because the contexts from which these functions are used mean that they can't fail (with the source of hashAlg currently being hardcoded inside ubuntu-core-fde-utils).